estimagic/differentiation/first_order_auxiliary.py

```diff
@@ -0,0 +1,37 @@
+"""
+This module contains auxiliary functions for a one step differentiation.
+"""
+
+
+def central(func, f_x0, params_sr, var, h, *func_args, **func_kwargs):
```

Comment:
please add docstrings to all functions, documenting all inputs and specifying to which functions the args and kwargs will be passed.